room/src/Revature.Room.Lib/Room.cs

```
@@ -0,0 +1,68 @@
+using System;
+
+namespace Revature.Room.Lib
+{
+  /// <summary>
+  /// Room class for holding basic information about a room temporarily
+  /// </summary>
+  public class Room
+  {
+    /// <summary>
+    /// Number of beds in a room, can also be interpreted as room's full capacity
+    /// </summary>
+    private int _numberOfBeds;
+
+    /// <summary>
+    /// Unique Identifier for each room, assigned by complex service
+    /// </summary>
+    public Guid RoomId { get; set; }
+
+    /// <summary>
+    /// Another way to uniquely identify a room, assigned by complex service
+    /// </summary>
+    public string RoomNumber { get; set; }
+
+    /// <summary>
+    /// Logic for checking that the Number of Beds is greater than zero
+    /// Server side validation
+    /// </summary>
+    public int NumberOfBeds
+    {
+      get => _numberOfBeds;
+      set
+      {
+        if (value > 0) _numberOfBeds = value;
```

Should throw an exception